### DISCUSSION TITLE
Rowy Pro Pricing

### DISCUSSION THREAD

**Discussion by joekman990** [AntlerVC/firetable#1272] [Support / Q&A] [answered]
I am trying to understand the pricing for Rowy Pro if i am the workspace owner and i want to add a user to my space but i don't need them to have a paid account is that possible?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @joekman990, you can invite your team to a base project for free. However, if the project you're inviting your users to is an upgraded PRO project you'll be priced at $12 per seat per month. For more information, check out: https://www.rowy.io/pricing or reach out to us on our Discord server in case of any queries.
